roles/thelounge/templates/config.js.j2

```diff
@@ -0,0 +1,472 @@
+"use strict";
+module.exports = {
```

Comment:
for a later PR: you could handle the whole config as a variable, to allow full customisation without having to define variables for each and every setting. Something like this:

```suggestion
{% with %}
{% set defaults =
 {
    'public': False,
    # 'host' not added as default is undefined
    ..,
  }
}%
{% set prefs =
  {
    'reverseProxy': (thelounge__reverse_proxy | bool),
    ...
  } |
  # Do not add the key if undefined.
  combine({'host': thelounge__listen_addr} if thelounge__listen_addr else {})
%}
{% set conf = defaults |
  combine(thelounge__config_extra, recursive=True) |
  combine(prefs, recursive=True) %}
module.exports = {{ conf | to_nice_json }}
{% endwith %}
```